interface/usergroup/usergroup_admin_add.php

```diff
@@ -13,7 +13,7 @@
 <head>
 
 <link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
-<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'];?>/bootstrap-3-3-4/dist/css/bootstrap.css" type="text/css">
```

Comment:
remember to add this:
``````
<?php if ($_SESSION['language_direction'] == 'rtl'): ?>
    <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'];?>/bootstrap-rtl-3-3-4/dist/css/bootstrap-rtl.css" type="text/css">
<?php endif; ?>

(note this will be short lived as fancybox is removed, so wouldn't worry about a class for this)